_posts/george_mihaiescu/2018-03-13-Quick-Kubernetes-installation.md

```diff
@@ -0,0 +1,333 @@
+~~~bash
+root@kubernetes:~# kubeadm token create --print-join-command --ttl 24h
+kubeadm join --token 123091.73f18d0e3afcd54b 172.16.16.15:6443 --discovery-token-ca-cert-hash sha256:000ccded7dfae148a89ed2bce9c17f584a7048e2e898da2cc102a11e14334f42
+~~~
+
+
+Associate a floating IP address to your instance so you can reach it in your browser:
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/floating.png" />
+</figure>
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/floating2.png" />
+</figure>
+
+
+In order to see the Grafana dashboard that was provisioned by the script, visit **http://floatingIP:port**, where floating IP is the public address you allocated to your Kubernetes master node, and the Grafana port is displayed in the instance's **console log** (example below):
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/console.png" />
+</figure>
+
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/Grafana_UI.png" />
+</figure>
+
+In order to see the Kubernetes dashboard that was provisioned by the script, visit **https://floatingIP:port**, where floating IP is the public address you allocated to your Kubernetes master node, and the Kubernetes dashboard allocated port is displayed in the instance's **console log** (example below):
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/kubernetes_dashboard.png" />
+</figure>
+
+When you visit that page, the browser will prompt you because the SSL cert is self-signed, but you'll have to go ahead and accept it.  
+Use the **Token** authentication method and paste the **token** you can find in the **console log**:
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/token.png" />
+</figure>
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/token_init.png" />
+</figure>
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/dashboard.png" />
+</figure>
+
+---
+
+**Kubernetes worker installation instructions (aka Kubelet).**
+
+
+
+Now that you have successfully deployed the Kubernetes master with its own dashboard and monitoring solution, you will need to deploy some worker nodes (aka Kubelets).  
+Follow the same steps as above to deploy one or more VMs that will be used to run the Kubernetes workloads.  
+
+Make sure the flavor you choose is large enough to accommodate whatever production workloads you plan running in your cluster, and **VERY IMPORTANT: change the "join" command at the end of the bash script below to use the valid token and IP address for your existing cluster** that you obtained from the Kubernetes master's console log:
+
+
+<figure>
+    <img src="{{site.urlimg}}george_mihaiescu/Kubernetes/join.png" />
+</figure>
+
+Dump this script in the Use the following cloud-init script while starting the VM(s):
```

Comment:
Add quotes around init-script name.

Dump this script in the "Use the following cloud-init script" while starting the VM(s):